### DISCUSSION TITLE
sympy.physics.mechanics. Definition of a point.

### DISCUSSION THREAD

**Discussion by Peter230655** [sympy/sympy#26340] [Q&A]
In a recent program, written by two emminent experts, I found this line:

_Bl = Bo.locatenew('B_l', -l/4*B.y)_

When I started to learn sympy.physics.mechanics a few years ago this was the way to do it.
But sometime later, if I remember correctly, it was said that one should do it like this:

_Bl = me.Point('Bl')
Bl.set_pos(Bo, -l/4*B.y)_

and that the 'old way' may be deprecated.  
Is my memory failing me? Is the 'old way' still acceptable?
Thanks for any help!

**Comment by moorepants** [comment]
If you look at the code for `locatenew` you can see it does the same thing as your second example:

https://github.com/sympy/sympy/blob/master/sympy/physics/vector/point.py#L280

There is no plan to deprecate `locatenew()`.

**Comment by Peter230655** [comment]
Thanks! Clear now,  both equally acceptable.
